libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -395,5 +435,66 @@ private void CheckErrors()
                 }
             }
         }
+
+        private string ExtractNameSpace(IList<string> options)
+        {
```

Comment:
We have tooo many operators on the split of "=" to get the key and value.
it is better to extract a new method `KeyValuePair<string, string> ExtractKeyValueOptions(string keyName)`

Reply:
refactor to a new method